### ISSUE TITLE
The lines (links) connecting the circles are shown in dev mode but not shown in build mode

### ISSUE THREAD

**Issue by sNishant011** [tsparticles/astro#2]
The lines (links) connecting the circles are shown in dev mode but not shown in build mode.
**In Development mode:**

![image](https://user-images.githubusercontent.com/49269819/228897811-543c948c-55af-4141-ad5e-eb327be80e2e.png)

**In Build mode:**

![image](https://user-images.githubusercontent.com/49269819/228898067-2235fdb3-5e14-407b-98ba-895734126ec0.png)

I am using `astro-particles`.

Here is my config:
```
{
  background: {
    color: {
      value: "#ffffff",
    },
  },
  fpsLimit: 60,
  fullScreen: {
    enable: true,
    zIndex: -1,
  },
  particles: {
    color: {
      value: "#74c043",
    },
    links: {
      color: "#74c043",
      distance: 150,
      enable: true,
      opacity: 0.5,
      width: 1,
    },
    collisions: {
      enable: true,
    },
    move: {
      direction: "none",
      enable: true,
      outMode: "bounce",
      random: false,
      speed: 2,
      straight: false,
    },
    number: {
      density: {
        enable: true,
        value_area: 800,
      },
      value: 80,
    },
    opacity: {
      value: 0.5,
    },
    shape: {
      type: "circle",
    },
    size: {
      random: true,
      value: 5,
    },
  },
}
```

**Code**
```astro

// ....

const coolParticles: ISourceOptions = {
  background: {
    color: {
      value: "#ffffff",
    },
  },
  fpsLimit: 60,
  fullScreen: {
    enable: true,
    zIndex: -1,
  },
  particles: {
    color: {
      value: "#74c043",
    },
    links: {
      color: "#74c043",
      distance: 150,
      enable: true,
      opacity: 0.5,
      width: 1,
    },
    collisions: {
      enable: true,
    },
    move: {
      direction: "none",
      enable: true,
      outMode: "bounce",
      random: false,
      speed: 2,
      straight: false,
    },
    number: {
      density: {
        enable: true,
        value_area: 800,
      },
      value: 80,
    },
    opacity: {
      value: 0.5,
    },
    shape: {
      type: "circle",
    },
    size: {
      random: true,
      value: 5,
    },
  },
};


<script>
  import { tsParticles } from "tsparticles-engine";
  import { loadFull } from "tsparticles";

  (async () => {
    await loadFull(tsParticles);
  })();
</script>

<Particles id="tsparticles" options={coolParticles} />

//...

```

**Comment by KenjiPcx** [comment]
I was able to replicate this too

**Comment by matteobruni** [comment]
Fixed in `astro-particles` v2.10.0, the README file is updated for this.
